src/main/resources/sql/script_29.03.2024.16_07.sql

```diff
@@ -135,24 +138,24 @@ CREATE TABLE `order`
     `tax`          DECIMAL,
     `shipping_id`  VARCHAR(36),
     `currency`     ENUM ('UAH','EUR','USD')            NOT NULL,
+    `status`       ENUM ('PENDING', 'PROCESSING', 'SHIPPED', 'DELIVERED', 'CANCELED'),
     `created_at`   TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
     FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
     FOREIGN KEY (`shipping_id`) REFERENCES `shipping_info` (`id`) ON DELETE SET NULL ON UPDATE CASCADE
 );
 
 CREATE TABLE `ordered_item`
 (
-    `id`       VARCHAR(36) PRIMARY KEY,
-    `order_id` VARCHAR(36)  NOT NULL,
-    `item_id`  VARCHAR(36),
-    `title`    VARCHAR(255) NOT NULL,
-    `size`     VARCHAR(255) NOT NULL,
-    `color`    VARCHAR(255) NOT NULL,
-    `price`    DECIMAL      NOT NULL,
-    `count`    INTEGER      NOT NULL,
-    `image`    VARCHAR(255) NOT NULL,
-    FOREIGN KEY (`order_id`) REFERENCES `order` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
-    FOREIGN KEY (`item_id`) REFERENCES `item` (`id`) ON DELETE SET NULL ON UPDATE CASCADE
+    `id`           VARCHAR(36) NOT NULL,
+    `item_id`      VARCHAR(36) NOT NULL,
+    `item_info_id` VARCHAR(36),
```

Comment:
I don't think it's necessary for us to save the item information ID in the ordered item.